.releaserc.js

```diff
@@ -3,8 +3,8 @@ module.exports = {
     '@semantic-release/commit-analyzer',
     '@semantic-release/release-notes-generator',
     '@semantic-release/changelog',
-    '@semantic-release/git',
     '@semantic-release/npm',
```

Comment:
The order matters here:
https://github.com/semantic-release/semantic-release/blob/master/docs/usage/plugins.md#plugins-declaration-and-execution-order

`npm` plugin updates the `package.json` so it need to be done before the `git` plugin hooks.